src/libraries/System.Linq.Parallel/tests/QueryOperators/OrderByThenByTests.cs

```diff
@@ -516,7 +516,7 @@ public static void OrderBy_ThreadedDeadlock(Labeled<ParallelQuery<int>> labeled,
         }
 
         // Heavily exercises OrderBy, but only throws one user delegate exception to simulate an occasional failure.
-        [Theory]
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
I'm surprised so few tests needed to be disabled.  I guess we're mostly saved by PLINQ's fall back to single-threaded execution when ProcessorCount == 1.

Reply:
Yeah I was surprised too and came to the same conclusion.